docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -241,8 +243,28 @@ The default is `false`.
 metrics-host="127.0.0.1"
 ```
 
-Specifies the host on which the [Prometheus](https://prometheus.io/) monitoring service accesses Pantheon
-metrics. The default is `127.0.0.1`. The metrics server respects the [`--host-whitelist` option](#host-whitelist).
+Specifies the host on which [Prometheus](https://prometheus.io/) accesses [Pantheon metrics](../Using-Pantheon/Debugging.md#monitor-node-performance-using-prometheus) or which 
+Pantheon pushes metrics to a [Prometheus Pushgateway](https://github.com/prometheus/pushgateway). 
+The metrics server respects the [`--host-whitelist` option](#host-whitelist).
+
+The default is `127.0.0.1`. 
+
+### metrics-mode
```

Comment:
Metrics mode has been removed.  it is repalced with `--metrics-push-enabled`.  I would recommend discussing it _after_ `--metrics-host` and `--metrics-port`.  `--metrics-enabled` only enables the prometheus polling and `--metrics-push-enabled` only enables prometheus push gateway support.  Both `enabled` flags cannot be specified as an error will be thrown.

Reply:
Done